arm-templates/ExampleWithProxySQL/template.json

```diff
@@ -0,0 +1,447 @@
+        "nestedtemplateMySQL": "[uri(variables('_artifactsLocation'), concat('nested/mysql.json', variables('_artifactsLocationSasToken')))]",
+        "nestedtemplateMySQLReplica": "[uri(variables('_artifactsLocation'), concat('nested/mysqlReplica.json', variables('_artifactsLocationSasToken')))]",
+        "sourceServerId": "[resourceId('Microsoft.DBforMySQL/servers', parameters('serverName'))]",
+        "skuName": "[concat(if(equals(parameters('skuTier'),'GeneralPurpose'),'GP',if(equals(parameters('skuTier'),'Basic'),'B',if(equals(parameters('skuTier'),'MemoryOptimized'),'MO',''))),'_',parameters('skuFamily'),'_',parameters('vCPU'))]",
+        "sku": {
+            "name": "[variables('skuName')]",
+            "tier": "[parameters('skuTier')]",
+            "capacity": "[parameters('vCPU')]",
+            "size": "[parameters('skuSizeMB')]",
+            "family": "[parameters('skuFamily')]"
+        }
+    },
+    "resources": [
+        {
+            "apiVersion": "2017-05-10",
+            "name": "MySQLServer",
+            "type": "Microsoft.Resources/deployments",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[variables('nestedtemplateMySQL')]"
+                },
+                "parameters": {
+                    "sku": {
+                        "value": "[variables('sku')]"
+                    },
+                    "serverName": {
+                        "value": "[parameters('serverName')]"
+                    },
+                    "location": {
+                        "value": "[variables('location')]"
+                    },
+                    "version": {
+                        "value": "[parameters('version')]"
+                    },
+                    "administratorLogin": {
+                        "value": "[parameters('administratorLogin')]"
+                    },
+                    "administratorLoginPassword": {
+                        "value": "[parameters('administratorLoginPassword')]"
+                    },
+                    "backupRetentionDays": {
+                        "value": "[parameters('backupRetentionDays')]"
+                    },
+                    "geoRedundantBackup": {
+                        "value": "[parameters('geoRedundantBackup')]"
+                    },
+                    "enableAzureResources": {
+                        "value": "[variables('enableAzureResources')]"
+                    },
+                    "ServerId": {
+                        "value": "[variables('sourceServerId')]"
+                    }
+                }
+            }
+        },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "MySQLServerReplicas",
+            "type": "Microsoft.Resources/deployments",
+            "condition": "[greater(variables('numberOfReplicas'),0)]",
+            "dependsOn": [
```

Comment:
Does ARM template support the delay? Like, create the replica server 5 minutes later after the master is created.

Reply:
as far as I know, delay is not there.